modules/gitops_configuring_sensitive_annotations_in_the_argo_cd_web_ui.adoc

```diff
@@ -0,0 +1,43 @@
+// Module is included in the following assemblies:
+//
+// * securing_openshift_gitops/masking-sensitive-annotations-in-the-argo-cd-web-ui.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops_configuring_sensitive_annotations_in_the_argo_cd_web_ui_{context}"]
+= Configuring sensitive annotations in the Argo CD Web UI
+
+Configure sensitive annotations in the Argo CD Web UI by performing the following steps.
+
+.Procedure
+
+. Log in to the {OCP} web console. 
+
+. In the *Administrator* perspective of the web console, click *Operators* -> *Installed Operators*.
+
+. Create or select the project where you want to install the user-defined Argo CD instance from the *Project* list.
+
+. Select *{gitops-title}* from the installed Operators list and click the *Argo CD* tab.
+
+. Edit your ArgoCD CR by completing the following steps:
+.. Under the `spec.extraConfig` section, add the `resource.sensitive.mask.annotations` key.
+.. Specify the annotation keys to mask as a comma-separated list.
+
++
+[source,yaml]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: example
+spec:
+  extraConfig:
+    resource.sensitive.mask.annotations: openshift.io/token-secret.value, api-key, token #<1>
+----
+<1> - Specify comma-separated list of sensitive annotation values such as, `api-key` and `token`.
```

Comment:
```suggestion
<1> - Specify comma-separated list of sensitive annotation values such as, `openshift.io/token-secret.value`, `api-key` and `token`.
```
shall we also mention `openshift.io/token-secret.value` ?

Reply:
Fixed.